ingest/CreateArrayImportTsvs.wdl

```diff
@@ -4,10 +4,11 @@ workflow CreateArrayImportTsvs {
 
   input {
     File input_vcf
-    String? probe_info_table
-    String? probe_info_file
+    File? probe_info_table
+    File? probe_info_file
```

Comment:
shouldn't probe_info_table still be a string?

Reply:
yup, good catch